lib/plugins/helper/tagcloud.js

```diff
@@ -57,14 +58,15 @@ function tagcloudHelper(tags, options) {
     const ratio = length ? sizes.indexOf(tag.length) / length : 0;
     const size = min + ((max - min) * ratio);
     let style = `font-size: ${parseFloat(size.toFixed(2))}${unit};`;
+    const name = className ? ` class="${className}-${Math.round(ratio * 10)}"` : '';
```

Comment:
I perfer `ceil`

if use `round`
| Level | 0 | 1 | ... | 9 | 10 |
| --- | --- | --- | --- | --- | --- |
| Probability | 5% | 10% | --- | 10% | 5% |

if use `ceil`
| Level | 1 | 2 | ... | 9 | 10 |
| --- | --- | --- | --- | --- | --- |
| Probability | 10% | 10% | --- | 10% | 10% |

Reply:
But how about level 0? `Math.ceil(0) == 0`, but there is no level 0 in your `Math.ceil` table.
Your theory seems to describe the case where tag.length approaches infinity
| Level | 0 | 1 | 2 | ... | 9 | 10 |
| --- | --- | --- | --- | --- | --- | --- |
| Probability | 0% | 10% | 10% | ... | 10% | 10% |

However, the actual situation is that tag.length is finite (and ratio is discrete), so the probability of level 0 must be greater than 0.

Reply:
```suggestion
    let level = Math.ceil(ratio * 10);
    if (level === 0) level ++;
    const name = className ? ` class="${className}-${level}"` : '';
```

Reply:
In fact the asymmetry of level 0 and level 10 comes from this line
```js
const ratio = length ? sizes.indexOf(tag.length) / length : 0;
```

```js
const ratio = length ? (sizes.indexOf(tag.length) + 0.5) / length : 0;
```
Of course, I don't think this change is necessary. We are designing a blog system instead of some sort of scientific computing. Such an error (0.5 / length) is acceptable.

Reply:
`name` could be replaced with `attr`.